canvg.js

```diff
@@ -46,8 +46,8 @@
 			for (var i=0; i<svgTags.length; i++) {
 				var svgTag = svgTags[i];
 				var c = document.createElement('canvas');
-				c.width = svgTag.clientWidth;
-				c.height = svgTag.clientHeight;
+				c.width = svgTag.clientWidth || svgTag.getBoundingClientRect()['width'];
+				c.height = svgTag.clientHeight || svgTag.getBoundingClientRect()['height'];

```

Comment:
this is fine to take care of the FF bug (similar to comment in https://bugzilla.mozilla.org/show_bug.cgi?id=874811)

i might prefer it to only be called once so something like the following insteaad
```js
c.width = svgTag.clientWidth;
c.height = svgTag.clientHeight;
if (!c.width && !c.height) {
  var rect = svgTag.getBoundingClientRect();
  c.width = rect.width;
  c.height = rect.height;
}
```